aws/cloud-formation/cache/cache.yaml

```diff
@@ -61,13 +61,13 @@ Resources:
         - Key: Project
           Value: !Ref Project
   
-  CacheCluster:
+  CacheClusterV2:
```

Comment:
Onkos tää välttämätöntä uudelleen nimetä?

Reply:
AWS ohjeistaa vaihtamaan koko instanssi. Varmaan saman efektin sais muullakin tavalla. Nyt vedin näin. Uudelleen nimeämisen avulla AWS kuvittelee, että instantsi poistetaan ja luodaan uudestaan.